src/components/layout/header/navbar/Navbar.tsx

```diff
@@ -6,13 +6,14 @@ import { useLocation } from 'react-router-dom';
 
 const Navbar = ({ isOpen }: INavbarProps) => {
     const location = useLocation();
-
     return (
         <Wrapper isOpen={isOpen}>
             {LINKS.map((link, index) => (
-                <NavLink href={link.linkTo} key={index} active={location.pathname === link.linkTo}>
-                    {link.text}
-                </NavLink>
+                <li key={index} >
```

Comment:
maybe Navlink could cosist of the li tag so there is no need to add it everywhere like this